spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraJoinRDD.scala

```diff
@@ -47,6 +49,11 @@ class CassandraJoinRDD[L, R] private[connector](
 
   override protected val classTag = rightClassTag
 
+  override protected[connector] lazy val rowReader: RowReader[R] = manualRowReader match {
+    case Some(rowReader) => rowReader
```

Comment:
yeah, just rename the pattern matching param - actually the param name can be shorter in this case, as its scope of visibility is very narrow